tuftool/src/error.rs

```diff
@@ -1,6 +1,8 @@
 // Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: MIT OR Apache-2.0
 
+// Not really worried about the memory penalty of large enum variants here
+#![allow(clippy::large_enum_variant)]
```

Comment:
What makes it large?  Nothing particularly stands out to me, and it seems smaller than it did before...

Reply:
Clippy is angry that some of the enum variants are larger than others. I really think it's overly pedantic in this case and for a CLI tool I don't think we care about this.
```
warning: large size difference between variants
   --> tuftool/src/error.rs:145:5
    |
145 | /     LinkTarget {
146 | |         indir: PathBuf,
147 | |         outdir: PathBuf,
148 | |         source: tough::error::Error,
149 | |         backtrace: Backtrace,
150 | |     },
    | |_____^
```